### PR TITLE
Explicitly telling the user what their account is.

### PR DESCRIPTION
**Pull request by Silver-Golden**
Added a little bit of text letting the user know to use username@example.com and that it is not an email address.

### CODE REVIEW

File: lib/views/signup-success.html [hunk 1]
```diff
@@ -1,3 +1,8 @@
 <h2>You&rsquo;ve signed up</h2>
 
-<p>Thanks, <em><%= params.username %></em>.</p>
+<p>Thanks, <em><%= params.username %></em>.
+  <br />
+  Your storage account account is <%= params.username %>@<%= host %>
```
Comment (raucao) [inline lib/views/signup-success.html:5]: Typo?

Comment (Silver-Golden) [reply lib/views/signup-success.html:5]: Ah teh double account, ill fix that

**Comment by AngeloR** [comment]
Just wanted to follow up on this! It's definitely something that would be useful as a new user to RS and armadietto!

**Comment by raucao** [comment]
Closing in favor of #81. (Added you as co-author to the commit, @Silver-Golden.)

**Pull request closed without merging**